docs/source/quickstart.mdx

```diff
@@ -0,0 +1,405 @@
+  description: String @join__field(graph: PRODUCTS)
+  price: Price @join__field(graph: PRODUCTS)
+  salesRank(category: ProductCategory = ALL): Int @join__field(graph: PRODUCTS)
+  salesRankOverall: Int @join__field(graph: PRODUCTS)
+  salesRankInCategory: Int @join__field(graph: PRODUCTS)
+  category: ProductCategory @join__field(graph: PRODUCTS)
+  images(size: Int = 1000): [String] @join__field(graph: PRODUCTS)
+  primaryImage(size: Int = 1000): String @join__field(graph: PRODUCTS)
+  reviews: [Review] @join__field(graph: REVIEWS)
+  reviewSummary: ReviewSummary @join__field(graph: REVIEWS)
+}
+
+enum ProductCategory {
+  ALL
+  GIFT_CARDS
+  ELECTRONICS
+  CAMERA_N_PHOTO
+  VIDEO_GAMES
+  BOOKS
+  CLOTHING
+}
+
+type Query {
+  bestSellers(category: ProductCategory = ALL): [Product] @join__field(graph: PRODUCTS)
+  categories: [Department] @join__field(graph: PRODUCTS)
+  product(id: ID!): Product @join__field(graph: PRODUCTS)
+}
+
+"""
+This is an Entity, docs:https://www.apollographql.com/docs/federation/entities/
+You will need to define a __resolveReference resolver for the type you define, docs: https://www.apollographql.com/docs/federation/entities/#resolving
+"""
+type Review
+  @join__owner(graph: REVIEWS)
+  @join__type(graph: REVIEWS, key: "id")
+{
+  id: ID! @join__field(graph: REVIEWS)
+  rating: Float @join__field(graph: REVIEWS)
+  content: String @join__field(graph: REVIEWS)
+}
+
+type ReviewSummary {
+  totalReviews: Int
+  averageRating: Float
+}
+```
+
+</ExpansionPanel>
+
+As you can see, this composed schema includes all of the types and fields from our subgraph schemas, along with many additional directives that the gateway uses to support our federated architecture.
+
+Similar to what we did with our subgraph schemas, now append ` > supergraph.graphql` to the above command to write the composed schema to a file:
+
+```shell
+rover supergraph compose --config ./supergraph.yaml > supergraph.graphql
+```
+
+## 4. Start the gateway
+
+We can now edit our `index.js` file to pull in our composed schema. Replace the file's contents with the following:
+
+```js:title=index.js
+const { ApolloServer } = require('apollo-server');
+const { ApolloGateway } = require('@apollo/gateway');
+const { readFileSync } = require('fs');
+
+const supergraphSchema = readFileSync('./supergraph.graphql').toString();
+
+const gateway = new ApolloGateway({
+  supergraphSdl: supergraphSchema
+});
+
+const server = new ApolloServer({
+  gateway,
+  // Subscriptions are not currently supported in Apollo Federation
+  subscriptions: false
+});
+
+server.listen().then(({ url }) => {
+  console.log(`🚀 Gateway ready at ${url}`);
+}).catch(err => {console.error(err)});
+```
+
+Now with our `supergraphSchema` properly populated, let's start up the gateway again with `node index.js`. This time, there's no error!
+
+We can quickly open our browser to `localhost:4000` to view our composed schema in GraphQL Playground:
```

Comment:
Maybe this is a good opportunity to introduce people to dev graphs?

Reply:
I agree that we should find a way to get the Explorer in earlier soon, but it's a tricky proposition as it requires account creation, and we're currently able to get away with _no_ account creation in part 1 (which I believe widens the top of the funnel).

I can revisit this post-Summit